runtime2/src/main/kotlin/edu/cornell/cs/apl/viaduct/runtime/Runtime.kt

```diff
@@ -14,6 +16,8 @@ interface Runtime {
     suspend fun <T> receive(type: KType, sender: Host): T
 
     suspend fun <T> send(type: KType, value: T, receiver: Host)
+
+    suspend fun url(host: Host): URL
```

Comment:
why is this a URL? shouldn't it be just an IP address?

Reply:
We need a URL for TLS. URLs can be IP if necessary.